psd-web/app/helpers/investigations_helper.rb

```diff
@@ -248,6 +248,34 @@ def suggested_previous_owners
     all_past_owners || []
   end
 
+  def source_details_rows(investigation, viewing_user)
+    rows = [
+      investigation.should_display_date_received? ? { key: { text: "Received date" }, value: { text: investigation.date_received.to_s(:govuk) } } : nil,
+      investigation.should_display_received_by? ? { key: { text: "Received by" }, value: { text: investigation.received_type.upcase_first } } : nil,
+      { key: { text: "Source type" }, value: { text: investigation.complainant.complainant_type } },
+      { key: { text: "Contact details" }, value: { text: investigation.contact_details_for_display(viewing_user) } }
+    ]
+
+    rows.compact!
+    rows
+  end
+
+  # rubocop:disable Rails/OutputSafety
+  def team_list_html(investigation)
+    return investigation.teams_with_access.first.name if investigation.teams_with_access.length == 1
+
+    tag.ul(class: "govuk-list") do
+      investigation.teams_with_access.map { |team| tag.li(escape_once(team.name)) }.join.html_safe
+    end
```

Comment:
Suggest using `concat` to simplify this, and avoid having to explicitly escape and mark as `html_safe`: 

```suggestion
    tag.ul(class: "govuk-list") do
      investigation.teams_with_access do |team|
        concat tag.li(team.name)
      end
    end
```